cypress/support/commands/case-commands.js

```diff
@@ -90,9 +90,11 @@ function addSubcase(type, newShortTitle, longTitle, step, stepName) {
   cy.get('.vlc-input-field-block').should('have.length', 5);
 
   //Set the type
-  cy.get('.vlc-input-field-block').eq(0).within(() => {
-    cy.contains(type).scrollIntoView().click();
-  });
+  if(type) {
+    cy.get('.vlc-input-field-block').eq(0).within(() => {
+      cy.contains(type).scrollIntoView().click();
+    });
+  }
 
```

Comment:
Breken bovenstaande changes de bestaande implemenaties van addSubcase niet?

Reply:
Nee, wanneer type gezet is, gaat hij er gewoon doorgaan zoals hij vroeger deed..